pkg/acl/acl.go

```diff
@@ -315,10 +316,23 @@ func CreateAclRequestDataToAclData(data *AclRequestDataWithError) cpkafkarestv3.
 }
 
 func PrintACLsFromKafkaRestResponse(cmd *cobra.Command, acls []cckafkarestv3.AclData) error {
+	all, err := cmd.Flags().GetBool("all")
+	if err != nil {
+		return err
+	}
+
 	list := output.NewList(cmd)
 	for _, acl := range acls {
+		principal := acl.GetPrincipal()
+		if !all {
+			if integerId, err := principalHasIntegerId(principal); err != nil {
```

Comment:
Let's make this clear it's a boolean
```suggestion
			if hasIntegerId, err := principalHasIntegerId(principal); err != nil {
```